src/codec/CMakeLists.txt

```diff
@@ -50,4 +50,4 @@ set_property(TARGET ${CODEC_TARGET} PROPERTY FOLDER "Serlio Maya Plugin")
 
 ### setup install target
 
-install(TARGETS ${CODEC_TARGET} DESTINATION ${INSTALL_FOLDER_PREFIX}/plug-ins/ext)
+install(TARGETS ${CODEC_TARGET} RUNTIME DESTINATION ${INSTALL_FOLDER_PREFIX}/plug-ins/ext)
```

Comment:
Looking good. Just to be sure: the addition of RUNTIME makes `install` ignore the .lib files, right?

Reply:
true, why can't the cmake guys not abstract this distinction away by default??